### PR TITLE
Updated doc for Console Logger, added prefix option

### PR DESCRIPTION
**Pull request by harshjain2**
Updated doc for Console Logger, added prefix option

### CODE REVIEW

File: src/vstest.console/Resources/Resources.resx [hunk 1]
```diff
@@ -238,6 +238,9 @@
 
       Change the verbosity level for console logger. Allowed values for verbosity: quiet, minimal and normal.
       Example: /logger:console;verbosity=&lt;Defaults to "minimal"&gt;
+
```
Comment (codito) [inline src/vstest.console/Resources/Resources.resx]: Please provide a link back to the https://github.com/Microsoft/vstest-docs/blob/master/docs/report.md#1-console-logger

**Comment by harshjain2** [comment]
Snapshot :
image

File: src/vstest.console/Resources/Resources.resx (hunk 1 above)
Comment (Faizan2304) [reply src/vstest.console/Resources/Resources.resx]: We should also update syntax and example in https://github.com/Microsoft/vstest-docs/blob/master/docs/report.md#1-console-logger to include prefix.

**Comment by harshjain2** [comment]
@dotnet-bot test this please.

**Review by Faizan2304 — APPROVED** [review]

File: src/vstest.console/Resources/Resources.resx [hunk 2]
```diff
@@ -236,8 +236,9 @@
       Visual Studio Test Results File (TRX) use /logger:trx [;LogFileName=&lt;Defaults to unique file name&gt;]
       Creates file in TestResults directory with given LogFileName.
 
-      Change the verbosity level for console logger. Allowed values for verbosity: quiet, minimal and normal.
-      Example: /logger:console;verbosity=&lt;Defaults to "minimal"&gt;
+      Change the verbosity level and add the verbosity level as prefix in log messages for console logger. Allowed values for verbosity: quiet, minimal and normal.
```
Comment (Faizan2304) [inline src/vstest.console/Resources/Resources.resx]: nit: The file Resources.Designer.cs should not get updated?

**Review by Faizan2304 — APPROVED** [review]

File: src/vstest.console/Resources/Resources.resx (hunk 2 above)
Comment (singhsarab) [inline src/vstest.console/Resources/Resources.resx]: This string does not make sense for me.
It is a bit confusing, how can I enable/disable adding prefix is not clear

Comment (harshjain2) [reply src/vstest.console/Resources/Resources.resx]: Suggestions are welcome.

Comment (singhsarab) [reply src/vstest.console/Resources/Resources.resx]: @pvlakshm

**Comment by pvlakshm** [comment]
```
      Specify a logger for test results. For example, to log results into a 
      Visual Studio Test Results File (TRX) use /logger:trx[;LogFileName=&lt;Defaults to unique file name&gt;]
      Creates file in TestResults directory with given LogFileName.

      Change the verbosity level in log messages for console logger as shown below
      Example: /logger:console;verbosity=&lt;Defaults to "minimal"&gt
      Allowed values for verbosity: quiet, minimal and normal.

      Change the diagnostic level prefix for console logger as shown below
      Example: /logger:console;prefix=&lt;Defaults to "false"&gt
      More info on Console Logger here : https://aka.ms/console-logger

      To publish test results to Team Foundation Server, use TfsPublisher as shown below
      Example: /logger:TfsPublisher;
                Collection=&lt;team project collection url&gt;;
                BuildName=&lt;build name&gt;;
                TeamProject=&lt;team project name&gt;
                [;Platform=&lt;Defaults to "Any CPU"&gt;]
                [;Flavor=&lt;Defaults to "Debug"&gt;]
                [;RunTitle=&lt;title&gt;]</value>

**Review by singhsarab — APPROVED** [review]

**Comment by singhsarab** [comment]
Cherry-pick into rel.15.5

**Pull request merged**